Description: Skip broken test
  On recent minitest, Test::Unit is not defined by default. Pending: check if
  upstream is affected by this, with all the bundle ocus pocus.
Author: Antonio Terceiro <terceiro@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2015-09-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_command_guesser.rb
+++ b/test/test_command_guesser.rb
@@ -15,5 +15,4 @@ class TestCommandGuesser < Minitest::Tes
   should_guess_command_name "Integration Tests", "/some/path/test/integration/foo_bar_controller_test.rb", "test/integration/**/*_test.rb"
   should_guess_command_name "Cucumber Features", "features", "cucumber", "cucumber features"
   should_guess_command_name "RSpec", "/some/path/spec/foo.rb"
-  should_guess_command_name "Unit Tests", "some_arbitrary_command with arguments" # Because Test::Unit const is defined!
 end if SimpleCov.usable?
